### PR TITLE
Maker API / block updates

### PR DESCRIPTION
**Pull request by bcjordan**
- pixels -> array of colorLeds
- piezo -> buzzer
- remove led methods except for on/off
- add palette param names for low-level maker methods
# Before

image

image

image
# After

image

image

### CODE REVIEW

File: apps/src/makerlab/dropletConfig.js [hunk 1]
```diff
@@ -18,30 +18,35 @@ const playSongConfig = {
...
+  {func: 'led', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
+  {func: 'led.on', category: 'Circuit'},
+  {func: 'led.off', category: 'Circuit'},
 
-  {func: 'piezo', category: 'Circuit', type: 'readonlyproperty', noAutocomplete: true},
-  {func: 'piezo.frequency', category: 'Circuit', params: ['500', '100']},
-  {func: 'piezo.note', category: 'Circuit', params: ['"A4"', '100']},
-  {func: 'piezo.stop', category: 'Circuit'},
-  {func: 'piezo.play', category: 'Circuit', paletteParams: ['song'], params: [JSON.stringify(playSongConfig)]},
+  {func: 'on', blockPrefix: colorLedBlockPrefix, category: 'Circuit', tipPrefix: pixelType, modeOptionName: '*.on', objectDropdown: {options: colorPixelVariables}},
```
Comment (bcjordan) [inline apps/src/makerlab/dropletConfig.js:36]: (Note: array accessor blocks aren't yet drop-down-able, so the `objectDropdown` property is just a placeholder.)

File: apps/src/makerlab/dropletConfig.js [hunk 2]
```diff
@@ -18,30 +18,35 @@ const playSongConfig = {
   tempo: 100000
 };
 
-const pixelType = '[Pixel]';
-const colorPixelVariables = ['pixels', 'pixel0', 'pixel1', 'pixel2', 'pixel3', 'pixel4', 'pixel5', 'pixel6', 'pixel7', 'pixel8', 'pixel9'];
-const simpleLedVariables = ['led', ...colorPixelVariables];
+const pixelType = '[ColorLed].';
+const colorPixelVariables = Array.from({length: 10}, (_, index) => `colorLeds[${index}]`);
```
Comment (Bjvanminnen) [inline apps/src/makerlab/dropletConfig.js:22]: could also be done as `_.range(10).map(index => colorLeds[${index}]` which I personally find marginally easier to read than Array.from({length: 10})

**Comment by Bjvanminnen** [comment]
lgtm

File: apps/src/makerlab/dropletConfig.js (hunk 2 above)
Comment (bcjordan) [inline apps/src/makerlab/dropletConfig.js:22]: I like that better, will change.

**Pull request merged**